backoff/backoff.go

```diff
@@ -0,0 +1,71 @@
+package backoff
+
+import (
+	"time"
+)
+
+type RandomGenerator interface {
+	Int63n(n int64) int64
+}
+
+type Config struct {
```

Comment:
We should add documentation for the the config properties.

Reply:
Added documentation.